src/app/shared/pipes/author-list.pipe.ts

```diff
@@ -1,18 +1,25 @@
 import { Pipe } from "@angular/core";
-import { mockedAuthorsList } from "../mocks/mocks";
+import { CoursesStoreService } from "@app/services/courses-store.service";
+import { map } from "rxjs";
 
 @Pipe({
   name: "authorList",
+  pure: false,
```

Comment:
Ideally, pipes should be pure. Making it impure is an option, but not desired. Can you please refactor this to make it independent of the courseService?
Not the in the component template files you can use multiple pipes, even with async ones!